### ISSUE TITLE
Log the index name when a field_stat filter query raises an exception

### ISSUE THREAD

**Issue by matthagenbuch** [elastic/curator#1200]
### Expected Behavior
Sometimes, due to misconfiguration or bad data in Elasticsearch, the "age" filtertype with "field_stats" source may raise an exception.  In my case, the exception was caused by an index with no documents, resulting in the field_stats aggregation returning no min/max values for the designated field.  When this happens, the system should log the name of the index which resulted in the failed aggregation query so that the user can troubleshoot the issue.

### Actual Behavior
The system logs the exception,  but not the index which was being processed at the time:
```
13:33:03,954 DEBUG          curator.indexlist _get_field_stats_dates:263  RESPONSE: {'took': 7, 'timed_out': False, '_shards': {'total': 5, 'successful': 5, 'skipped': 0, 'failed': 0}, 'hits': {'total': 0, 'max_score': 0.0, 'hits': []}, 'aggregations': {'min': {'value': None}, 'max': {'value': None}}}
13:33:03,954 DEBUG          curator.indexlist _get_field_stats_dates:267  r: {'min': {'value': None}, 'max': {'value': None}}
13:33:03,954 ERROR                curator.cli                    run:184  Failed to complete action: close.  <class 'TypeError'>: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

### Steps to Reproduce the Problem
1.  Create an Elasticsearch index named `my-empty-index` with no documents
2. Configure a Curator action using an age/field_stats filter like:
```yaml
actions:
  1:
    action: delete_indices
    description: "Delete old indices"
    options:
      timeout_override: 300
      disable_action: False
    filters:
      - filtertype: pattern
        kind: prefix
        value: my-empty-index
      - filtertype: age
        direction: older
        source: field_stats
        field: '@timestamp'
        stats_result: max_value
        unit: days
        unit_count: 2
```
3. Run the curator action

### Specifications

  - Version: 5.4.1
  - Platform: Mac OSX
  - Subsystem: ???

## Context (Environment)
I'm using a field_stats age filter to select indices for closing/deletion in a production ES cluster.  Sometimes empty indexes are created due to an unrelated process which match the other filters in my filter list.  When this happens, the Curator action fails because the field_stats aggregation returns no values.

## Detailed Description
I propose, at a minimum, that the system should log the name of the index being processed when this type of error occurs.  Even better, it would be nice to have a filter option to ignore indexes that fail the aggregation query for one reason or another.

**Comment by untergeek** [comment]
> In my case, the exception was caused by an index with no documents, 

This was already addressed in #1167 and is in the latest version of Curator.

**Comment by matthagenbuch** [comment]
My bad, I didn't realize a newer version was released.  This can be closed
